CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.4.0 (TBD)
 
+### Behavior Changes
+
+- Aggregate function results (from `DataFrame.agg`, `DataFrame.describe`, etc) no longer strips away non-printing characters from column names.
```

Comment:
Would something like this work?

- Aggregate function results, from functions such as `DataFrame.agg` and `DataFrame.describe`, no longer strip away non-printing characters from column names.